cdap-ui/app/cdap/components/PluginJSONCreator/Create/Content/WidgetCollection/WidgetInput/index.tsx

```diff
@@ -0,0 +1,113 @@
+/*
+ * Copyright © 2020 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import withStyles, { StyleRules, WithStyles } from '@material-ui/core/styles/withStyles';
+import If from 'components/If';
+import { WIDGET_TYPES } from 'components/PluginJSONCreator/constants';
+import PluginInput from 'components/PluginJSONCreator/Create/Content/PluginInput';
+import { ICreateContext } from 'components/PluginJSONCreator/CreateContextConnect';
+import * as React from 'react';
+
+const styles = (): StyleRules => {
+  return {
+    widgetInput: {
+      '& > *': {
```

Comment:
Thats a little dangerous and takes away the essential use of css-in-js. This one component controls every single element rendered within itself. Will introduce issues when things change in unexpected places. Can we reconsider this?

Reply:
I assigned `widgetInput` to each child instead, so that it has the same styling without  using `& > *'`.

Reply:
changed!